docs/philosophy.md

```diff
@@ -0,0 +1,25 @@
+# Our philosophy
+
+## Why open-wc
+
+- The goal of Open Web Components is to empower everyone with a powerful and battle-tested setup for creating and sharing open source web components.
+- Javascript fatigue: Enjoy the peace of mind that comes from having a well-known default solution for almost everything. Open-wc has got you covered.
+- Making things accessible and approachable for people: Easy to use, ready to use, 'it just works' feeling
+
+## Why webcomponents
+
+- keep it frank, and friendly (no throwing shade at frameworks)
+	- close to browser standards etc
+	- bundle size?
```

Comment:
Have you ever asked, "what if I could only subtract to solve problems?" What if you could *remove code*, *gain performance,* and *streamline implementation* across everything you build? Web components provide this through the power of subtraction.

* Replace javascript you'd normally need to include by letting the browser handle things [links to relevant browser standards]
* Get access to blazingly fast performance [link to benchmarks]
* Write using a streamlined component API that combines the best bits of React, Vue, and Vanilla JS!

Reply:
I like this. I'd drop the first sentence ("have you ever asked") and `s/Get access to blazingly fast/Increase your app's/g`

Reply:
I just now added a `Why Web Components` section. This would be perfect for the paragraph just under that header.